### ISSUE TITLE
[Bug] "Encrypting & Uploading 100.00%" Pop-up stuck

### ISSUE THREAD

**Issue by elendil7** [cryptee/web-client#199]
## Describe the bug

When switching documents while the current document is still in proccess of being uploaded to the cloud, a persistent "Encrypting & Uploading (100.00%)" pop-up is displayed at the bottom right corner of the screen. It does not get removed until you refresh the entire page, no matter how many documents you switch to.

## Steps to reproduce the behavior
1. Go to any document
2. While that document is uploading, click on another document (to load it).
3. The "Encrypting & Uploading 100.00%" pop-up should not be permanently stuck in the bottom right hand side of the screen.

## Expected behavior

I expect the pop-up window to automatically close, after switching documents.

## Screenshots
![image](https://github.com/cryptee/web-client/assets/80429382/b0125eb0-dfda-4f4f-8e40-63c27beb0ca7)

## System Information (please complete the following information):
 - Device: Computer
 - OS: Windows 10 Pro
 - Browser: Firefox (hardened)
 - Browser: Version Latest
 - Browser Extensions / Add-ons / Content/Ad-Blockers: Ublock origin, Umatrix, Decentraleyes
 - Do you have any DNS-based content/ad-blockers?: Not applicable
 - Are you on a managed network?: Not applicable

**Comment by johnozbay** [comment]
Hi there 👋🏻

Many million thanks for this detailed bug description. Thanks to your help we managed to identify and reproduce this bug quickly. 

This is now fixed this in the alpha version and the issue is linked to our internal issue/release tracker.
Once the update with this fix is out (should be latest in a few days), this issue will be closed automatically, and you'll get a notification!

Thank you so much for filing this, your help and patience!

All the best,

J
